reader/processor/processor.go

```diff
@@ -41,10 +41,6 @@ func ProcessFeedEntries(store *storage.Storage, feed *model.Feed) {
 	for _, entry := range feed.Entries {
 		logger.Debug("[Processor] Processing entry %q from feed %q", entry.URL, feed.FeedURL)
 
-		if isBlockedEntry(feed, entry) || !isAllowedEntry(feed, entry) {
```

Comment:
Why did you moved this condition below? The goal is to avoid scraping a website if not necessary.

I'm assuming you want to filter based on the content scraped. In this case, I would add the content filtering part after the scraping part.

Reply:
Hello, sorry for this, I just want to make sure that the entries are completed before they are filtered. But this does make extra scraping procedure when 
1. Scraping is enabled
2. Entry's title hits the blocking rule

or
1. Scraping is enabled
2. This option `apply_filter_to_content` is not enabled
3. Entry's title does not hit the keeping rule

I have tried, but I can't find a simple way to deal with it. Sorry. Do you have any suggestions?